test/tutorial/cpp/asyncfinish/arraysum.cpp

```diff
@@ -0,0 +1,55 @@
+#include "hclib_cpp.h"
+#include <sys/time.h>
+
+int SIZE=10000000; //10M
+int* array;
+
+// TIMING HELPER FUNCTIONS
+long get_usecs ()
```

Comment:
We already have hclib_current_time_ns() and hclib_current_time_ms(). Could you replace all timer utilities in the application code to use one of these HClib functions?